src/integration/pages/articles/mundo/__snapshots__/amp.test.js.snap

```diff
@@ -87,11 +87,75 @@ Object {
 
 exports[`AMP Articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/mundo"`;
 
+exports[`AMP Articles Footer I can see a link: AdChoices / Do Not Sell My Info 1`] = `"https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/"`;
+
+exports[`AMP Articles Footer I can see a link: BBC News, Mundo 1`] = `"/mundo"`;
+
+exports[`AMP Articles Footer I can see a link: Contacta a la BBC 1`] = `"https://www.bbc.com/mundo/institucional-36400011"`;
+
+exports[`AMP Articles Footer I can see a link: Cookies 1`] = `"https://www.bbc.com/usingthebbc/cookies/"`;
+
+exports[`AMP Articles Footer I can see a link: Lee sobre nuestra postura acerca de enlaces externos. 1`] = `"https://www.bbc.co.uk/help/web/links/"`;
+
+exports[`AMP Articles Footer I can see a link: Política de privacidad 1`] = `"https://www.bbc.com/mundo/institucional-36400009"`;
+
+exports[`AMP Articles Footer I can see a link: Por qué puedes confiar en la BBC 1`] = `"https://www.bbc.com/mundo/institucional-51359666"`;
+
+exports[`AMP Articles Footer I can see a link: Sobre la BBC 1`] = `"https://www.bbc.com/mundo/institucional-36400007"`;
+
+exports[`AMP Articles Footer I can see a link: Términos de uso 1`] = `"https://www.bbc.com/mundo/institucional-36400005"`;
+
 exports[`AMP Articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. La BBC no se hace responsable del contenido de sitios externos. Lee sobre nuestra postura acerca de enlaces externos."`;
 
-exports[`AMP Articles Header I can see the branding 1`] = `"BBC News, Mundo"`;
+exports[`AMP Articles Header I can see a navigation link: América Latina 1`] = `"/mundo/america_latina"`;
+
+exports[`AMP Articles Header I can see a navigation link: América Latina 2`] = `"/mundo/america_latina"`;
+
+exports[`AMP Articles Header I can see a navigation link: Centroamérica Cuenta 1`] = `"/mundo/noticias-43826245"`;
+
+exports[`AMP Articles Header I can see a navigation link: Centroamérica Cuenta 2`] = `"/mundo/noticias-43826245"`;
```

Comment:
why is there 2 snapshots for each link?

Reply:
There are hidden nav items (used for the burger menu at lower resolutions). I struggled to find a way to exclude one, but I'll have another look.

Reply:
ah ok. I wonder if a more specific query selector would help